### PR TITLE
Installer Script

### PR DESCRIPTION
**Pull request by ParthKolekar**
Added installer / run script for ubuntu

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+_install() {
+	if [ ! -e /usr/bin/saathi ];
+	then
+		ln -s `readlink -f $0` /usr/bin/saathi
+		if [ ! $? -eq 0 ];
+		then
+			echo "${YELLOW}Re-run as sudo to setup this script${NC}"
+		fi
+		apt-get install lamp-server^ # install the whole goddammed lamp stack
```
Comment (nisargjhaveri) [inline setup.sh:15]: We are just using php and mysql.. Why the whole lamp stack?

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	else
+		echo "${RED}Config not Found.. Please copy the config file and make changes.${NC}"
+		exit 1
+	fi
+}
+
+_start () {
+	_check
+
+	php -S localhost:8000 > /dev/null &
```
Comment (nisargjhaveri) [inline setup.sh:32]: If we remove `> /dev/null &` we won't need stop script at all. Just `ctrl+c` to stop the server. And also it'll show status in the terminal all the time (which is preferred 'personally' :P)

File: setup.sh (hunk 1 above)
Comment (ParthKolekar) [inline setup.sh:15]: Too lazy to write down exact dependencies.

File: setup.sh (hunk 2 above)
Comment (ParthKolekar) [inline setup.sh:32]: My personal prefrence differs.

**Comment by nisargjhaveri** [comment]
Fair enough.. merging.

Also, we should add a warning that this involves some changes in root owned directories.

**Pull request merged**